lightning/src/ln/onion_utils.rs

```diff
@@ -2817,6 +2817,27 @@ fn process_failure_packet(
 	update_attribution_data(onion_error, shared_secret, hold_time);
 }
 
+pub(crate) fn process_fulfill_attribution_data(
```

Comment:
Can we add a comment for this method, including that it's for intermediate/final nodes?